tests/ast/decoder.py

```diff
@@ -828,11 +853,32 @@ def decode_expr(self, expr: proto.Expr) -> Any:
                         val,
                     )
                 return val
+
             case "sp_dataframe_show":
                 df = self.decode_expr(
                     self.symbol_table[expr.sp_dataframe_show.id.bitfield1][1]
                 )
                 return df.show()
+
+            case "sp_relational_grouped_dataframe_builtin":
+                grouped_df = self.decode_expr(
+                    expr.sp_relational_grouped_dataframe_builtin.grouped_df
+                )
+                cols = [
+                    self.decode_expr(arg)
+                    for arg in expr.sp_relational_grouped_dataframe_builtin.cols.args
+                ]
+                agg_name = expr.sp_relational_grouped_dataframe_builtin.agg_name
+                if expr.sp_relational_grouped_dataframe_builtin.cols.variadic:
```

Comment:
Both arms of the "if" are doing the same thing ?

Reply:
derp ignore..

Reply:
Nice catch. Needs `*agg_name` in the first case.

Reply:
cols is not being used - intentional ?

Reply:
I'm working on this -- it should be used here!